gsd-web/src/boot/buildVersion.js

```diff
@@ -0,0 +1,13 @@
+// Uses code from https://stackoverflow.com/a/34518749
+
+const fs = require('fs');
+
+const rev = fs.readFileSync('../.git/HEAD').toString().trim();
```

Comment:
I'm guessing what happens here is that we end up baking this into the dist files upon compilation, meaning that we don't have to worry about production not necessarily having the git meta alongside the dist.

Reply:
I am seeing that this fails on build for some reason. So I am going to close this for now and find a new solution,